ui/tutorial-renderer/src/main/java/org/cru/godtools/tutorial/layout/TipsLayout.kt

```diff
@@ -24,18 +29,27 @@ import com.airbnb.lottie.compose.rememberLottieComposition
 import org.cru.godtools.base.ui.theme.GodToolsTheme
 import org.cru.godtools.tutorial.R
 
-@Preview(showBackground = true)
 @Composable
-internal fun TipsTutorialLayout() = GodToolsTheme() {
+internal fun TipsTutorialLayout(
```

Comment:
let's also pass in a `page: Page` parameter for this Compose method. you will have access to the current `Page` object from where you call `TipsTutorialLayout()`